package.json

```diff
@@ -7,6 +7,7 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
+    "test:e2e": "playwright test"
```

Comment:
Can't add ',' here
It gives me the error "Trailing comma"

Reply:
that's because during merge conflict you removed all the incoming changes.

Pls keep the incoming changes related to commitlint here and also below in devDependencies

If you can't figure it out then let me know, I can do that from my side

Reply:
Please do it for me, I can't seem to figure it out.